persistence/src/test/scala/hmda/persistence/institutions/InstitutionPersistenceSpec.scala

```diff
@@ -25,9 +27,25 @@ class InstitutionPersistenceSpec extends ActorSpec {
     }
 
     "get institution by id" in {
-      val institution = DemoData.testInstitutions.head
-      probe.send(institutionsActor, GetInstitution(institution.id))
-      probe.expectMsg(Some(institution))
+      probe.send(institutionsActor, GetInstitutionById(headInst.id))
+      probe.expectMsg(Some(headInst))
+    }
+
+    "get institutions by id" in {
+      val ids = DemoData.testInstitutions.map(i => i.id).toList
+      probe.send(institutionsActor, GetInstitutionsById(ids))
+      probe.expectMsg(DemoData.testInstitutions)
+    }
+
+    "get institution by respondent id" in {
+      val respondentId = headInst.respondent.externalId.value
+      probe.send(institutionsActor, GetInstitutionByRespondentId(respondentId))
+      probe.expectMsg(headInst)
+    }
+
+    "find institution by email domain" in {
+      probe.send(institutionsActor, FindInstitutionByDomain("bank0.com"))
+      probe.expectMsg(Set(headInst))
     }
 
```

Comment:
Could we get the un-happy path specs copied over from the `InstitutionViewSpec` as well? A few I noticed: one for `GetInstitutionsById` when the request is for ids that don't exist, and one for `FindInstitutionByDomain`, when the domain doesn't exist.